test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Visitors/DictionaryObjectTypeVisitorTests.cs

```diff
@@ -34,6 +34,7 @@ public void Init()
         }
 
         [DataTestMethod]
+        [DataRow(typeof(Dictionary<object, string[]>), false)]
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.